cmd/deploy.go

```diff
@@ -218,14 +216,8 @@ func watchReleaseCommand(ctx context.Context, cc *cmdctx.CmdContext, apiClient *
 	g, ctx := errgroup.WithContext(ctx)
 	interactive := cc.IO.IsInteractive()
 
-	s := spinner.New(spinner.CharSets[11], 100*time.Millisecond)
-	s.Writer = os.Stderr
-	s.Prefix = "Running release task..."
-
-	if interactive {
-		s.Start()
-		defer s.Stop()
-	}
+	cc.IO.StartProgressIndicatorMsg("Running release task...")
```

Comment:
Aren't we using the new deploy command now? So this should be done in watch.go?

Reply:
It's done in both places.

Reply:
OOOPS

Reply:
Ah yeah, other commands still depend on the old deploy.